obspy/core/inventory/response.py

```diff
@@ -961,6 +961,51 @@ def is_close(a, b):
 
         return sampling_rates
 
+    @classmethod
+    def get_flat_response(cls, sensitivity=1., frequency=1.,
+                          input_units="M/S", output_units="M/S",
+                          include_pole_zero_stage=False):
```

Comment:
What's the reasoning behind not always having at least one stage? We can't even `Response.plot()` it without any stages..

```
/home/megies/git/obspy/obspy/core/inventory/response.py in _call_eval_resp_for_frequencies(self, frequencies, output, start_stage, end_stage, hide_sensitivity_mismatch_warning)
   1151             msg = ("Can not use evalresp on response with no response "
   1152                    "stages.")
-> 1153             raise ObsPyException(msg)
   1154 
   1155         import obspy.signal.evrespwrapper as ew

ObsPyException: Can not use evalresp on response with no response stages.
```

Reply:
This should now be fixed.  For Response.plot() it still requires the sampling_rate as we don't have any digital stages in the response that would indicate the sampling_rate.  However, the following works.
Response.plot(0.1, sampling_rate=1)